icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/NumberFormatDataDrivenTest.java

```diff
@@ -718,7 +718,7 @@ public void TestDataDrivenJDK() {
         // #13373: Since not all JDK implementations are the same, test only whitelisted JDKs
         // with known behavior. The JDK version should be occasionally updated.
```

Comment:
This test runs only on one version, right? -->
```suggestion
        // #13373: Since not all JDK implementations are the same, test only on a JDK
        // with known behavior. The JDK version should be occasionally updated.
```

On other JDKs/versions, does it fail? Or does `assumeTrue()` somehow exit this test function?

Reply:
I think that is how assumeTrue works; the test shows as skipped in JUnit out